### ISSUE TITLE
Conflict when more then one and is being used.

### ISSUE THREAD

**Issue by mceyzar** [caffaware/AIR-ANEs#1]
When more then one ANE is used in a project, it errors with:
   - Error occurred while packaging the application:
   - Conflict in file name : airlibrary-default/airlibrary-default.swc
Maybe better to keep the default library name different for each ANE

Regards.

**Comment by caffaware** [comment]
You are right. Will fix this asap

**Comment by mceyzar** [comment]
Any plans on open sourcing it so we as community can help fix things ?
I know you used to have them, as there are many forums with links to your website that is no longer there.

**Comment by caffaware** [comment]
Yup, you are right, I will upload the sources probably tonight. I currently don't have any time to update the ANEs

**Comment by caffaware** [comment]
All sources are online now, enjoy :-) 

https://github.com/caffaware/TouchID-ANE
https://github.com/caffaware/Pedometer-ANE
https://github.com/caffaware/iBeacon-ANE
